public/app/pipeline-orchestrator-worklet-init.js

```
@@ -0,0 +1,294 @@
+      if (!this._inputSAB || !this._outputSAB) return;
+      const orch = window._pipelineOrchestrator || window._vipApp?._orchestrator;
+      const mlWorker = orch?.mlWorker || window._vipApp?.mlWorker;
+      if (mlWorker) {
+        mlWorker.postMessage({
+          type:      'setSABs',
+          inputSAB:  this._inputSAB,
+          outputSAB: this._outputSAB,
+        });
+      } else {
+        // ml-worker not yet available — retry once it's assigned
+        const retryId = setInterval(() => {
```

ESLint

[error] 'setInterval' is not defined.

(no-undef)